backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/nuernberg/utils/WriteCsv.kt

```diff
@@ -29,12 +29,14 @@ fun writeCsvWithGeoInformation(input: List<AcceptingStore>) {
             "Telefon",
             "Email",
             "Website",
-            "Rabatt",
+            "RabattDE",
+            "RabattEN",
             "Kategorie"
         )
     )
 
     input.forEach { record ->
+        val discounts = splitDiscount(record.discount)
```

Comment:
Alternatively, I would remove this split function and only populate the "DE" version below for now and add a comment. The `splitDiscount` function seems to hacky for me :D

```kt
// For now, we do not properly support multi-language discount descriptions.
// Hence, the English and German translations of the discount were merged into the `discount` field
// of the `AcceptingStore` object in the `MapFromCsv` step.
// Therefore, we do not populate the CSV column for the english version, as splitting it back up is unreliable.
```

Reply:
Hm but we provide the updated csv (added geoinformation) to the customer on nextcloud

Reply:
Hmm, I see... So every quartal Nürnberg takes the CSV files that we upload to nextcloud, adjusts it, and sends it back to us?